ckanext/push_errors/tests/test_logging.py

```diff
@@ -0,0 +1,129 @@
+import requests
+import logging
+from datetime import datetime
+from unittest.mock import patch
+from ckanext.push_errors.logging import push_message
+from unittest.mock import ANY
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def test_push_message_with_valid_config(mock_config):
+    """Prueba el envío de mensajes con una configuración válida."""
+    # Mockear datetime.now para que sea consistente
+    fixed_time = datetime(2025, 1, 23, 14, 6, 38)
+    with patch("ckanext.push_errors.logging.datetime") as mock_datetime, \
```

Comment:
Better use `@patch` outside the function to avoid multiple `with` commands

Somethig like
```py
@patch("ckanext.push_errors.logging.datetime")
@patch("ckanext.push_errors.logging.requests.post")
def test_push_message_with_valid_config(self, mock_post, mock_datetime):
    ...
```

Reply:
?